src/shapes/polyline.class.ts

```diff
@@ -202,17 +216,30 @@ export class Polyline extends FabricObject {
    *
    * @private
    */
-  _getTransformedDimensions(options: Partial<TProjectStrokeOnPointsOptions>) {
-    return this.exactBoundingBox
-      ? super._getTransformedDimensions({
-          ...(options || {}),
-          // disable stroke bbox calculations
-          strokeWidth: 0,
-          // disable skewing bbox calculations
-          skewX: 0,
-          skewY: 0,
-        })
-      : super._getTransformedDimensions(options);
+  _getTransformedDimensions(options: any = {}) {
+    if (this.exactBoundingBox) {
+      let size: Point;
+      if (
+        Object.keys(options).some((key) =>
+          strokeProjectionOptions.includes(
+            key as keyof TProjectStrokeOnPointsOptions
+          )
+        )
+      ) {
+        const { width, height } = this._calcDimensions(options);
+        size = new Point(options.width ?? width, options.height ?? height);
+      } else {
+        size = new Point(
+          options.width ?? this.width,
+          options.height ?? this.height
+        );
+      }
+      return size.multiply(
+        new Point(options.scaleX || this.scaleX, options.scaleY || this.scaleY)
+      );
```

Comment:
Is this right? If the `strokeUniform = true` we shouldn't we add `strokeWidth`?

Reply:
Is it not safer to call `_getTransformedDimensions` with the options? 
That way, any change in logic there, we would take advantage of here too.

Reply:
I get your point and normally would agree but feels like the divergence of the calculation is so big we should detach from super